packages/ui/src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactText } from 'react';
+import { Badge as ChakraBadge, BadgeProps as ChakraBadgeProps, forwardRef, HTMLChakraProps } from '@chakra-ui/react';
+
+type BadgeVariant = 'solid' | 'outline' | 'subtle';
+
+type BadgeSize = 'small' | 'large';
+
+type BadgeColor = 'default' | 'brand' | 'blue' | 'green' | 'orange' | 'red';
+
+export interface BadgeProps extends HTMLChakraProps<'div'> {
+  children: ReactText | ReactText[];
+  /**
+   * Style variant of the badge
+   */
+  variant?: BadgeVariant;
```

Comment:
The badge shouldn't receive variant property. We don't know if we will use other variants of this component in the future so this property adds unnecessary complexity.